web/app/scripts/src/providers/project-config.es.js

```diff
@@ -0,0 +1,22 @@
+const ProjectConfig = function () {
+    let settings = {
+        apiRoot: 'localhost:8000'
+    };
+
+    return {
+        $get: function () {
+            return {
+                getApiRoot: () => settings.apiRoot
+            }
+        },
+        setApiRoot: (value) => {
+            settings.apiRoot = value;
+        }
+    };
+}
+
+export const provider = [
```

Comment:
Well but it doesn't work so well when you want to use that module in a different module, you rely on other modules anyway. So yeah it's basically the same pattern we've done before, but it's just an angular thing that you have to rely on actual strings